src/core/appdir.cpp

```diff
@@ -8,6 +8,8 @@
 #include <CImg.h>
 #include <fnmatch.h>
 #include <subprocess.hpp>
+#include <linuxdeploy/core/appdir.h>
```

Comment:
This is a local header in this case, and it has in fact been included already (see below).